tracpro/polls/views.py

```diff
@@ -600,7 +600,10 @@ def render_to_response(self, context, **response_kwargs):
                 questions = self.derive_questions().values()
 
                 resp_headers = ['Date']
-                contact_headers = ['Name', 'URN', 'Region', 'Group']
+                # FIXME: Should we change the column name "Region" to "Panel" and possibly break
+                # existing integrations that work by exporting .CSVs?
+                # Or leave it as "Region", and confuse people trying to create new integrations?
```

Comment:
The short answer is that I'm not concerned about existing integrations (I doubt that there are any to worry about). I would rather not confuse people, I would suggest we use the new name here.